tabbycat/options/presets.py

```diff
@@ -1,5 +1,7 @@
 import logging
 
+from django.utils.translation import gettext as _
```

Comment:
This should be `gettext_lazy`, otherwise all of the class attributes won't translate, because the functions for class attributes are evaluated when the class is loaded, not at runtime. (Let me know if you have questions about this.)